src/main/resources/config/application.yml

```diff
@@ -20,6 +19,8 @@ server:
     mime-types: application/json,application/xml,text/html,text/xml,text/plain
 
 revocation:
+  authorization:
+    reload: 600000
```

Comment:
@danieldelhoyo yes, I also see that it's a manual process, but I would say that reloading once a minute is fine and certainly decreases human's wait time.